app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
Mluvíš o teoretické změně. Implementace se týká aktuálního zadání a ne něčeho co může přijít v budoucnu. Samozřejmě implementace by měla být dostatečně flexibilní, aby nebyl potřeba velký přepis, proto není někdy radno přistupovat k velké abstrakci. V tomto případě je abstrakce celkem malá a určitě by byl přínos větší než copy-paste kód, kde v případě změn můžeš snadno zapomenout, co už jsi upravil a co ne.